gradle/libs.versions.toml

```diff
@@ -114,7 +119,7 @@ tink = { module = "com.google.crypto.tink:tink-android", version.ref = "tink" }
 coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "coroutines" }
 
 kotlin-bom = "org.jetbrains.kotlin:kotlin-bom:1.8.0"
-compose-bom = "androidx.compose:compose-bom:2023.05.01"
+compose-bom = "androidx.compose:compose-bom:2024.06.00"
```

Comment:
We actually had to upgrade to `2024.08.00` recently due to an unrelated bug, so this change probably isn't needed anymore/might cause a conflict.

Reply:
Resolved!